5.0 Draft/IDD/IDDs Orchestration System/eu.arrowhead.orchestration-pull-http-json.yml

```diff
@@ -0,0 +1,303 @@
+openapi: 3.0.3
+info:
+  title: Orchestration-pull service API description 
+  description: |
+    An API for using the orchestration-pull service of the Orchestration core system. This service
+    uses the orchestration store policies to recommend service providers to the requesting consumer to fulfil 
+    its task. 
+
+    The consumer system request for orchestration either by specifying the service definition that the consumer requires or 
+    by using stored SoS configurations or Orchestration rules if no service information is defined. 
+
+    _SoS configurations_ represents the connections needed in a SoS for it to perform its designed task. This means, describing what services
+    must each of the systems consume to perform their individual tasks. A SoS configuration can be seen as a series of _Orchestration rules_.
+    _Orchestration rules_ represents a desired interaction between systems consuming specific services from specific preferred providers. 
+
+    In the context of this document, we reffer to the Orchestration consumer as _requesting-system_ where _system-id_ is the
+    unique identifier that represents that system. The _requested-service_ represents the service information that the _requesting-system_
+    requires orchestration. In case the _requested_service_ is specified, the requires service definition must be included in the form, 
+    in addition, at least one interface must be specified. The orchestration system will attempt to match the _requested-service_ information
+    with services present on the orchestration rules or SoS configurations. 
+
+version: 5.0.0-rc.1
+
+# Additional information present in the IDD template
+x-transfer-protocol: [HTTP, HTTPS]
+x-encryption-method: to-add
+x-encoding: JSON
+x-compression: to-add
```

Comment:
Encoding and compression is something that is seamlessly taken care of by the HTTP layer. The only thing that could be of value to specify here are encodings and compression methods that _must_ be supported by implementations of this service.

I do believe that adding data like this to IDDs is a good idea, but I don't see it as a requirement for version 5.0. To reduce scope, I'd say all the `x`-fields should go away for now, including those about transfer protocols, encryption, semantics and ontology.